pkg/network/network_linux.go

```diff
@@ -283,7 +283,7 @@ func (m *Setup) SetArgs(args []string) error {
 				if len(ports) != 1 && len(ports) != 2 {
 					return fmt.Errorf("portmap port argument is badly formatted")
 				}
-				if n, err := strconv.ParseInt(ports[0], 0, 16); err == nil {
+				if n, err := strconv.ParseUint(ports[0], 0, 16); err == nil {
```

Comment:
looks good, line 288 and 297 cannot happen now, i guess.

Reply:
I left those in for now, so that just incase the `ParseUint()` gets reverted somehow, we still have an error check in place.

Reply:
imo useless checks should be removed in this PR.
How `ParseUint()` can be reverted?

Reply:
or better check that port is in range 0-65k and fail if not. If something gets reverted - it will still be correct.

Reply:
I was going to make it a `uint16` across the board since that fits the port range nicely. But in the Golang unix.SockaddrInet4 function, the port is an *int* value. So... that just breaks that idea, so I need to keep letting it use the lines like:
```
pm.HostPort = int(n)
```

As for getting reverting, it's _possible_ that a merge could have the old `ParseInt()` code. So, when it gets merged, it reverts it back to the incorrect way. Made more likely if someone force pushes.